ansible/playbooks/aix.yml

```diff
@@ -13,6 +13,7 @@
     PERL_MM_OPT: "INSTALL_BASE=/home/{{ Jenkins_Username }}"
   vars:
     swap_size: 4096
+    bootjdk: hotspot
```

Comment:
Should load this from adoptopenjdk_variables.yml.

Reply:
`bootjdk: hotspot` removed from `aix.yml` and added to `adoptopenjdk_variables.yml`